.github/workflows/build-docker.yml

```diff
@@ -143,8 +143,8 @@ jobs:
         # The final command becomes `docker buildx imagetools create -t tag1 -t tag2 ... <RUFF_BASE_IMG>@sha256:<sha256_1> <RUFF_BASE_IMG>@sha256:<sha256_2> ...`
         run: |
           docker buildx imagetools create \
-            $(jq -cr '.tags | map("-t " + .) | join(" ")' <<< "$DOCKER_METADATA_OUTPUT_JSON") \
-            $(printf '${RUFF_BASE_IMG}@sha256:%s ' *)
+            "$(jq -cr '.tags | map("-t " + .) | join(" ")' <<< "$DOCKER_METADATA_OUTPUT_JSON")" \
```

Comment:
Are the nested double quotes here not an issue because they're inside of a `$( )` construct?  I would have naively assumed you'd need to backslash-escape them.

Reply:
I think no because `$( )` should start a new context but I think in this specific case I should revert because of the glob.